src/main/java/org/dwcj/interfaces/IAttributable.java

```diff
@@ -0,0 +1,21 @@
+package org.dwcj.interfaces;
+
+public interface IAttributable {
```

Comment:
These adjectives sound unneutral. We should come up with something more  neutral. for instance:

- `HasAttribute` 
- `SupportsAttribute `

This apply of other interfaces too.